plugins/sslutils.c

```diff
@@ -35,6 +35,8 @@ static SSL_CTX *c=NULL;
 static SSL *s=NULL;
 static int initialized=0;
 
+int check_hostname = 0;
```

Comment:
I would prefer not to have more global variables

Reply:
Fair enough, I just copied it from nagios-plugins.
I see 3 possibilities:
1: add check_hostname as argument to np_net_ssl_init_with_hostname_version_and_cert.
  As far as I can see, only check_http is using that function
2: create an new function in sslutils.c
3: make np_net_ssl_init_with_hostname_version_and_cert variadic

Which one do you prefer ? I have a slight preference to 1, since also other checks might be interested in that check.